test/config.json

```diff
@@ -202,11 +202,11 @@
 			"File": "",
 			"Args": [],
 			"Command": [
-				"make",
-				"unit_test"
+				"travis/unit_test_runner.sh"
```

Comment:
Can you move this to tools/unit_test_runner.sh? It's going to be used any time someone does "make test" or "go run test.go", so it'll be confusing to have it live in travis/.